### PR TITLE
Add how-to for using integrated registry

### PR DESCRIPTION
**Pull request by simu**
## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
+= How to use the OpenShift integrated container registry
+
+The OpenShift integrated registry of each {product} zone is exposed outside the cluster at `registry.<zone>.appuio.cloud`.
+
+== Prerequisites
+
+To pull from the registry, your account needs the `registry-viewer` role.
+To push to the registry, your account needs the `registry-editor` role.
+These permissions are generally project-scoped.
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc:9]:
```suggestion
These permissions are generally namespace-scoped.
```

**Review by tobru — APPROVED** [review]

**Comment by simu** [comment]
https://github.com/appuio/component-openshift4-registry/pull/9 needs to be merged and rolled out on the APPUiO Cloud zones with appropriate config before we can merge this PR

File: docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+== Accessing the registry with your {product} user
+
+You can access the registry with your {product} credentials with the following steps:
+
+. Login to the desired zone and create docker credentials for the zone registry
++
+[source,bash]
+----
+zone=cloudscale-lpg-0 <1>
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc:23]:
```suggestion
zone=cloudscale-lpg-2 <1>
```

Just to be future proove.

File: docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc [hunk 3]
```diff
@@ -0,0 +1,85 @@
+= How to use the OpenShift integrated container registry
+
+The OpenShift integrated registry of each {product} zone is exposed outside the cluster at `registry.<zone>.appuio.cloud`.
+
+== Prerequisites
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc:5]: Unless changes were be made, a user will be admin in a namespace and thus can pull and push images. Can you please add that statement and mark the following as advanced/"in detail"?

**Review by corvus-ch — APPROVED** [review]
Basically LGTM.

If you can spare a bit more time, please extend the prerequisite a bit. Make it obvious that by default, a user has permission to pull and push images.

File: docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc (hunk 2 above)
Comment (simu) [reply docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc:23]: Do we care more about future-proofing the docs or about providing docs that actually work when copy-pasting? If we want future-proof docs, and don't care about copy-pastability we can directly do something like 

```suggestion
zone=ZONE-ID <1>
```

And adjust the callout to state that users need to replace `ZONE-ID` with an existing zone.

File: docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc (hunk 3 above)
Comment (simu) [reply docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc:5]: adjusted in the latest version

**Pull request merged**